python/ray/train/gbdt_trainer.py

```diff
@@ -24,18 +24,37 @@ def _convert_scaling_config_to_ray_params(
     ray_params_cls: Type["xgboost_ray.RayParams"],
     default_ray_params: Optional[Dict[str, Any]] = None,
 ) -> "xgboost_ray.RayParams":
-    default_ray_params = default_ray_params or {}
-    resources_per_worker = scaling_config.additional_resources_per_worker
-    num_workers = scaling_config.num_workers
-    cpus_per_worker = scaling_config.num_cpus_per_worker
-    gpus_per_worker = scaling_config.num_gpus_per_worker
+    ray_params_kwargs = default_ray_params.copy() or {}
+
+    resources = scaling_config.resources_per_worker or {}
+
+    cpus_per_actor = resources.pop("CPU", 0)
+    if not cpus_per_actor:
+        cpus_per_actor = ray_params_kwargs.get("cpus_per_actor", 0)
```

Comment:
ooc, why do we use `ray_params_kwargs` here, but use `default_ray_params` for other args?

Reply:
Ah that was an oversight, thanks - fixed!